src/test/java/com/iexec/common/utils/FeignBuilderTest.java

```diff
@@ -0,0 +1,112 @@
+package com.iexec.common.utils;
+
+import feign.Logger;
+import feign.RequestLine;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockserver.client.MockServerClient;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+/**
+ * The purpose of those tests is to check the {@code JacksonEncoder#encode}
+ * and {@code JacksonDecoder#decode} methods are properly overridden.
+ */
+public class FeignBuilderTest {
+
+    @Container
+    private static final MockServerContainer mockServer = new MockServerContainer(DockerImageName.parse("mockserver/mockserver"))
```

Comment:
New line after `=` so the line is not too long?

Reply:
Is it possible to use a Java-native mockServer instance? That would make tests lighter.

Reply:
Done in [0a15e8a](https://github.com/iExecBlockchainComputing/iexec-common/pull/307/commits/0a15e8a407d69ccad1176c7009c9b67fb6f3fdea)